apigw/src/shared/routes/auth/saml/index.ts

```diff
@@ -62,8 +63,13 @@ function createLoginHandler({
       req,
       'Login endpoint called'
     )
+    const locale = req.query.locale
+    const options: AuthenticateOptions = {
+      additionalParams: typeof locale === 'string' ? { locale } : {}
```

Comment:
More of a Q: will an incorrect locale trigger an error in `passport.authenticate` or will it fail silently? Just wondering whether we should explicitly catch that or not.

Reply:
A non-string locale (= not present, or uses one of the exotic query parameter formats supported by express) -> skipped
A locale unsupported by suomi.fi -> ???